docs/topics/Gateway_Events.md

```diff
@@ -420,7 +420,7 @@ The inner `d` key is a boolean that indicates whether the session may be resumab
 
 ### Auto Moderation
 
-All [Auto Moderation](#DOCS_RESOURCES_AUTO_MODERATION) related events are only sent to bot users which have the `MANAGE_GUILD` permission.
+All [Auto Moderation](#DOCS_RESOURCES_AUTO_MODERATION) related events are only sent to bot users with the `MANAGE_GUILD` permission.
```

Comment:
```suggestion
All [Auto Moderation](#DOCS_RESOURCES_AUTO_MODERATION) related events are only sent to bot users which have the `MANAGE_GUILD` permission.
```
This change is not related to this PR.

Reply:
"Which have" is unnecessarily wordy, so I changed it for consistency with what I added. This type of sentence only occurs twice in this document, including my addition. I'm willing to rewrite my sentence to be wordy, as well, if you prefer that.

Reply:
There's a third sentence following this format that also follows this wording:

> This event is only sent to bots with the `VIEW_AUDIT_LOG` permission.

Moreover, if you search in the entire repository, you'll only find one match for "which have the .+ permission" compared to "with the .+ permission," so this minor change feels warranted to me. Opening a second PR for it when making a related change feels overkill.

<img width="298" alt="image" src="https://user-images.githubusercontent.com/24176136/236407591-b8826f74-aea1-4286-b678-85df4dfa9320.png">

<img width="299" alt="image" src="https://user-images.githubusercontent.com/24176136/236407421-629b82b3-85f3-46f5-9ecb-072d9f36931a.png">

Reply:
Anyway, I'll accept your suggestion, since that should make most people satisfied. Thanks!

Reply:
It's not about the change itself. The change is not related to this PR. Open another PR for this changes.

Reply:
I think it's too minor of a change to warrant a separate PR.

Reply:
Well no. But I think this violates the contribution guidelines anyway.